packages/nova-types/src/nova-scheduling.interface.ts

```diff
@@ -0,0 +1,25 @@
+export interface NovaScheduling {
```

Comment:
Take a look at the comments on the other interface files - let's add some inline documentation around what this is and how to use it. At the moment, this is not clear on how a host would fulfill its duty to a component that is using this API.

Without a default implementation, it will also be difficult for component builders to use the API in their development environments - we will need to think through what the simplest path is for someone who wants to support this in Storybook for example...
/**
 * Describes the eventing contract that is used to communicate between UI components and their logical parent within a host application.
 *
 * Usage: components should create small independent packages that define the events they generate using this contract.
 * Logical parents can take a dependency on the event package, and use it to translate the events into appropriate actions within the larger application.
 */